content/guides/api-calls/permissions-and-errors/common-errors.md

```diff
@@ -113,6 +113,9 @@ for solution to common errors encountered when working with the Box APIs.
 | **Message**  | Item name too long                                                                                                                                                                                                                                                                                                                                                        |
 | **Solution** | Shorten the length of the name that is being supplied for the item. The maximum length of a file or folder name in Box is 255 characters or less.                                                                                                                                                                                                                         |
 |              |                                                                                                                                                                                                                                                                                                                                                                           |
+| **Error**    | `metadata_after_file_contents`                                                                                                                                                                                                                                                                                                                                            |
+| **Message**  | Metadata is included after file contents in a file upload request.                                                                                                                                                                                                                                                                                                        |
```

Comment:
This error seems weird. I didn't know you could apply Box Metadata at the same time as a file. Do we mean the `attributes` instead?

Reply:
Also how is this related to this PR?

Reply:
Ignore me, seen the PR comment